pontoon/base/views.py

```diff
@@ -334,6 +334,7 @@ def _serialize_translation_values(query):
                 "script": translation["locale__script"],
             },
             "translation": translation["string"],
+            "is_preferred": translation["locale__code"] in preferred_values,
```

Comment:
We could leave this out entirely if the value is `False` and hence reduce the response payload.